data/json/furniture_and_terrain/terrain-bridges-docks.json

```diff
@@ -216,5 +216,108 @@
     }
+  },
+  {
+    "type": "terrain",
+    "id": "t_metal_dock",
+    "name": "dock",
+    "description": "A metal platform held by a support made of concrete set into the ground.",
+    "symbol": "8",
+    "looks_like": "t_metal_floor",
+    "color": "gray",
+    "move_cost": 2,
+    "flags": [ "TRANSPARENT", "FLAT" ],
+    "deconstruct": { "ter_set": "t_water_sh", "items": [ { "item": "nail", "charges": [ 6, 12 ] }, { "item": "2x4", "count": 8 } ] },
+    "bash": {
+      "str_min": 20,
+      "str_max": 80,
+      "sound": "smash!",
+      "sound_fail": "whump!",
+      "ter_set": "t_water_moving_sh",
+      "items": [ { "item": "sheet_metal", "count": 2, "prob": 35 }, { "item": "scrap", "count": [ 2, 4 ] } ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_concrete_dock_deep_pile",
+    "name": "deep dock pile",
+    "description": "A sturdy pile made of poured concrete extending all the way to the riverbed.  A metal frame will be required to support a surface.",
+    "looks_like": "t_concrete_wall",
+    "symbol": "|",
+    "color": "gray",
+    "move_cost": 8,
+    "flags": [ "TRANSPARENT", "LIQUID", "SWIMMABLE", "DEEP_WATER", "FISHABLE" ],
+    "deconstruct": { "ter_set": "t_water_moving_dp", "items": [ { "item": "spike", "count": 4 }, { "item": "rock", "count": [ 6, 10 ] } ] },
+    "bash": {
+      "str_min": 50,
+      "str_max": 120,
+      "sound": "smash!",
+      "sound_fail": "whump!",
+      "ter_set": "t_water_moving_dp",
+      "items": [ { "item": "rock", "count": [ 12, 24 ] } ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_metal_dock_deep_frame",
+    "name": "deep dock frame",
+    "description": "A metal frame secured atop a concrete pile.  Adding a metal surface will make this into a proper dock section.",
+    "symbol": "#",
+    "color": "gray",
+    "move_cost": 6,
+    "flags": [ "TRANSPARENT", "NOITEM", "THIN_OBSTACLE", "MOUNTABLE", "TINY", "AUTO_WALL_SYMBOL", "UNSTABLE" ],
+    "deconstruct": { "ter_set": "t_concrete_dock_deep_pile", "items": [ { "item": "spike", "count": 8 }, { "item": "pipe", "count": 4 } ] },
+    "looks_like": "t_sconc_wall_halfway",
+    "bash": {
+      "str_min": 12,
+      "str_max": 100,
+      "sound": "clang!",
+      "sound_fail": "smash!",
+      "ter_set": "t_concrete_dock_deep_pile",
+      "items": [ { "item": "pipe", "count": 4, "prob": 25 }, { "item": "scrap", "count": [ 4, 8 ] } ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_metal_dock_deep",
+    "name": "deep dock",
+    "description": "A metal platform held by a tall metal pile that extends to the riverbed.  Very sturdy, and likely to outlast you.",
+    "symbol": "8",
+    "color": "gray",
+    "move_cost": 2,
+    "flags": [ "TRANSPARENT", "FLAT" ],
+    "deconstruct": {
+      "ter_set": "t_metal_dock_deep_frame",
+      "items": [ { "item": "nail", "charges": [ 12, 24 ] }, { "item": "sheet_metal", "count": 4 } ]
```

Comment:
Are the nails standing in for metal screws?

Reply:
yes, we don't have screws (unless added recently)